gemma-core/src/main/resources/ubic/gemma/core/loader/expression/singleCell/transform/requirements.txt

```diff
@@ -1,3 +1,4 @@
 anndata
 scipy
-numpy
\ No newline at end of file
+numpy<=2.2
+scanpy
```

Comment:
This is no longer necessary, all the tools for reading/writing MEX files are provided by Cell Ranger.